Description: Use Cardapio as the menu
 This patch simply replaces the MenuBar with Cardapio in the default layout
 This provides a better standard menu for Gnome users
 .
 consort-panel (1:3.4.2.1-3solusos3.2) two; urgency=low
 .
   * Set Cardapio as the default menu applet in the layout
Author: Ikey Doherty <ikey@solusos.com>

---
Reviewed-By: Ikey Doherty <ikey@solusos.com>
Last-Update: <2012-06-07>

--- consort-panel-3.4.2.1.orig/data/panel-default-layout.layout
+++ consort-panel-3.4.2.1/data/panel-default-layout.layout
@@ -4,8 +4,8 @@ orientation=bottom
 size=24
 y-bottom=0
 
-[Object menu-bar]
-object-iid=PanelInternalFactory::MenuBar
+[Object cardapio-menu]
+object-iid=cardapiognome3Factory::cardapiognome3
 toplevel-id=bottom-panel
 pack-type=start
 pack-index=0
